recipes/pyode/meta.yaml

```
@@ -0,0 +1,63 @@
+# defining package and version
+{% set name = "pyode" %}
+{% set version = "0.16" %}
+{% set soversion = "0.16.0" %}
+{% set sha256 = "4ba3b76f9c1314160de483b3db92b0569242a07452cbb25b368e75deb3cabf27" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/odedevs/ode/downloads/ode-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

As it's only used in one place I generally prefer not having a `sha256` variable and just writing it directly
```suggestion
  sha256: 4ba3b76f9c1314160de483b3db92b0569242a07452cbb25b368e75deb3cabf27
```